projects/aca-shared/src/lib/services/content-api.service.ts

```diff
@@ -361,4 +361,50 @@ export class ContentApiService {
   requestVersionDirectAccessUrl(nodeId: string, versionId: string): Observable<DirectAccessUrlEntry> {
     return from(this.versionsApi.requestDirectAccessUrl(nodeId, versionId));
   }
+
+  getNodeIcon(node: Node): string {
```

Comment:
this whole method is a wrapper for thumbnail service, it should be moved to thumbnail service instead

Reply:
Done.